R/checkSummations.R

```diff
@@ -105,12 +112,26 @@ checkSummations <- function(mifFile, outputDirectory = ".", template = NULL, sum
     # calculate differences for comparison
     comp <- comp %>%
       group_by(!!!syms(c("model", "scenario", "region", "period", "variable", "unit", "value"))) %>%
-      summarise(checkSum = sum(!!sym("childVal") * !!sym("factor")), .groups = "drop") %>%
+      summarise(checkSum = sum(!!sym("childVal") * !!sym("factor"), na.rm = TRUE),
+        summation = paste(
```

Comment:
"Summation" is a very generic, potentially misleading column. Can we agree on a better term, say "details" or "equation"?

Reply:
I can change for whatever name you prefer. Just let me know. I am usually quite bad at setting names for those things :)

Reply:
"details" works for me

Reply:
changed at https://github.com/pik-piam/piamInterfaces/pull/186/commits/1ffc05c2824672cdbddb206cc4e14ce96d690fdc